### ISSUE TITLE
Ambient mode bug (5.0.1)

### ISSUE THREAD

**Issue by eChrome5126** [dayanch96/YTLite#170]
### Describe the bug

```yaml
Disable ambient mode however it is not being disabled
```

### Steps to reproduce

```yaml
I disabled ambient mode in yt plus ‘player’ settings
Ambient mode still showing (left and right bars)
```

### Post Requisites

- [X] I have checked that a similar issue has not been reported before.
- [X] Verified the relevance of the project and the version you are using.
- [X] Provided a detailed description of the problem and have filled in all required fields.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.
- [X] By submitting this issue, **I confirm that I have filled in all the necessary fields and complied with the guidelines. Failure to comply may result in the issue being closed without discussion, and repeated violations may lead to restrict account from further participation.**

**Comment by VibePT** [comment]
Yes, it drives me nuts 😆

**Comment by dayanch96** [comment]
Fixed in 5.0.2
